crates/gpui/src/text_system/font_features.rs

```diff
@@ -58,7 +58,7 @@ impl<'de> serde::Deserialize<'de> for FontFeatures {
                 while let Some((key, value)) =
                     access.next_entry::<String, Option<FeatureValue>>()?
                 {
-                    if !is_valid_feature_tag(&key) {
+                    if key.len() != 4 || !key.chars().all(|c| c.is_ascii_alphanumeric()) {
```

Comment:
Let’s keep this pulled out in a separate function. I think it tells a better narrative.

Reply:
Fair enough!